pkgs/top-level/aliases.nix

```diff
@@ -294,6 +294,7 @@ mapAliases ({
   kinetic-cpp-client = throw "kinetic-cpp-client has been removed from nixpkgs, as it's abandoned."; # 2020-04-28
   kicad-with-packages3d = kicad; # added 2019-11-25
   krename-qt5 = krename; # added 2017-02-18
+  keyfinder = throw "keyfinder has been removed because it was broken"; # added 2021-02-01
```

Comment:
It seems to work for someone https://github.com/NixOS/nixpkgs/issues/18475#issuecomment-771504453

Reply:
False positive: https://github.com/NixOS/nixpkgs/issues/18475#issuecomment-772613652 ;-)

Reply:
@ajs124 Or would say we should keep it anyway? It would defiitely be marked as insecure.

Reply:
Instead of removing it, I now added a commit updating it to version 2.4.